src/Java/IO/PNGReader.java

```diff
@@ -0,0 +1,48 @@
+import vtk.vtkImageViewer2;
+import vtk.vtkPNGReader;
+import vtk.vtkNativeLibrary;
+import vtk.vtkRenderWindowInteractor;
+
+public class PNGReader 
+{
+  //-----------------------------------------------------------------
+  // Load VTK library and print which library was not properly loaded
+  static 
+  {
+    if (!vtkNativeLibrary.LoadAllNativeLibraries()) 
+    {
+      for (vtkNativeLibrary lib : vtkNativeLibrary.values()) 
+      {
+        if (!lib.IsLoaded()) 
+        {
+          System.out.println(lib.GetLibraryName() + " not loaded");
+        }
+      }
+    }
+    vtkNativeLibrary.DisableOutputWindow(null);
+  }
+  // -----------------------------------------------------------------
+
+  public static void main(String args[]) 
+  {
+    //parse command line arguments
+    if (args.length != 1) {
+      System.err.println("Usage: java -classpath ... Filename(.png) e.g Test.png");
+      return;
```

Comment:
Please change `Test.png` to `Gourds.png` and update the image. In that way people have access to an existing image just like the Cxx version.